### PR TITLE
Update stack.py [Python]

### LINKED ISSUE #125: Issue in stack
Standard implementation of stack involves only one pointer, which points at top of stack, It means that if you want to view all items of stack, you must remove all elements and then print them, displayStack() function violates the definition of standard stack.

In java also, there is no implementation of such function. However, if you print the stack, then it will print all the elements. But, as far as I know, it is a property of AbstractList class which is super super super class of Stack. I am aware that when we use inheritance, it inherits all the public properties. But, I guess this is unwanted inherited property. Also, If you visit wikipedia link there is no mention of such property of stack.

[https://en.wikipedia.org/wiki/Stack_(abstract_data_type)](url)

### PR DESCRIPTION
**Pull request by Prakash2403**
Fixes #125 

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous timespan of 7 days.

### CODE REVIEW

File: stack/stack.py [hunk 1]
```diff
@@ -3,10 +3,7 @@ def __init__(self):
         self._list = []
 
     def __str__(self):
-        return str(self._list)
-
-    def __len__(self):
-        return len(self._list)
+        raise Exception("Not allowed to view stack contents")
```
Comment (aviaryan) [inline stack/stack.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -3,7 +3,7 @@
         self._list = []
 
     def __str__(self):
-        raise Exception("Not allowed to view stack contents")	
+        raise Exception("Not allowed to view stack contents")
 
     def push(self, item):
         self._list.append(item)
```

Comment (aviaryan) [inline stack/stack.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -3,7 +3,7 @@
         self._list = []
 
     def __str__(self):
-        raise Exception("Not allowed to view stack contents")	
+        raise Exception("Not allowed to view stack contents")
 
     def push(self, item):
         self._list.append(item)
```

**Comment by Prakash2403** [comment]
@aviaryan @singhpratyush Any ideas why latest check is failed?

**Comment by aviaryan** [comment]
@Prakash2403 That looks to be some issue in gitmate configuration. It's a alpha stage software after all. I have done some change in it and I hope this won't happen again. 
For now, let's just ignore it. Sorry for the trouble.

File: stack/stack.py [hunk 2]
```diff
@@ -3,10 +3,7 @@ def __init__(self):
...
```
Comment (singhpratyush) [inline stack/stack.py]: You may simply remove this function from here. Throwing an exception on converting to string seems inappropriate.

File: stack/stack.py [hunk 3]
```diff
@@ -17,46 +14,26 @@ def pop(self):
...
-
 
 def main():
     st = Stack()
-
     # Push and print
     for i in range(10):
         st.push(i)
-        print(st)
+        print("Pushed ",i)
```
Comment (aashutoshrathi) [inline stack/stack.py]: add a whitespace after `,`

File: stack/stack.py (hunk 1 above)
Comment (aviaryan) [inline stack/stack.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -3,7 +3,7 @@
         self._list = []
 
     def __str__(self):
-        raise Exception("Not allowed to view stack contents")	
+        raise Exception("Not allowed to view stack contents")
 
     def push(self, item):
         self._list.append(item)
```

File: stack/stack.py [hunk 4]
```diff
@@ -17,46 +11,26 @@ def pop(self):
...
-        print('Exception: {0}'.format(e))
-
+    while True:  # Empty the stack
+        try:
+            st.pop()
+        except Exception as e:
+            print('Exception: {0}'.format(e))
+            break
 
 if __name__ == '__main__':
```
Comment (singhpratyush) [inline stack/stack.py]: Add two line space between top level members.

File: stack/stack.py [hunk 5]
```diff
@@ -17,46 +11,26 @@ def pop(self):
...
     main()
+
```
Comment (singhpratyush) [inline stack/stack.py]: Just single blank line in the end.

File: stack/stack.py (hunk 1 above)
Comment (aviaryan) [inline stack/stack.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -3,7 +3,7 @@
         self._list = []
 
     def __str__(self):
-        raise Exception("Not allowed to view stack contents")	
+        raise Exception("Not allowed to view stack contents")
 
     def push(self, item):
         self._list.append(item)
```

**Comment by aashutoshrathi** [comment]
Add newline at the end of file

**Comment by Prakash2403** [comment]
@aashutoshrathi newline is there at the end.

**Pull request merged**